tracer/tracer.go

```diff
@@ -133,7 +133,10 @@ func (t *Tracer) Flush() error {
 		return nil
 	}
 
-	_, err := t.transport.Send(spans)
+	// TODO[manu]: this doesn't make sense! change it with a proper buffer
```

Comment:
As we talked this morning, grouping spans with a common trace_id might just be enough. Might not be optimized, but will pass the check that is done downstream here: https://github.com/DataDog/raclette/blob/master/agent/receiver.go#L233